test_runner/regress/test_tenants.py

```diff
@@ -475,3 +476,37 @@ def only_int(samples: List[Sample]) -> int:
     assert counts
     log.info(f"directory counts: {counts}")
     assert counts[2] > COUNT_AT_LEAST_EXPECTED
+
+
+def test_tenants_parallel_creation(neon_simple_env: NeonEnv):
```

Comment:
why use neon_simple_env instead of neon_env_builder?

Reply:
why not? Do we need the builder?